source/layouts/layout.erb

```diff
@@ -51,111 +52,117 @@
-                      <li><a href="#postgresql-setup-with-django">PostgreSQL setup with Django</a></li>
-                    </ul>
-                  </li>
-                  <li><a href="#production-checklist">Production checklist</a></li>
-                </ul>
-              </li>
-
-              <li>
-                <a href="#deploying-backing-services">Deploying backing services</a>
-                <ul>
-                  <li><a href="#services-overview">Services overview</a></li>
-                  <li><a href="#paid-services">Paid services</a></li>
-                  <li><a href="#accessing-services">Accessing services</a></li>
-                  <li><a href="#future-services">Future services</a></li>
-                  <li>
-                    <a href="#using-a-postgresql-service">Using a PostgreSQL service</a>
-                    <ul>
-                      <li><a href="#creating-a-postgresql-service">Creating a PostgreSQL service</a></li>
-                      <li><a href="#accessing-postgresql-from-your-app">Accessing PostgreSQL from your app</a></li>
-                      <li><a href="#postgresql-service-maintenance-times">PostgreSQL service maintenance times</a></li>
-                      <li><a href="#postgresql-service-backup">PostgreSQL service backup</a></li>
-                      <li><a href="#high-availability-details">High availability details</a></li>
-                      <li><a href="#read-replicas">Read replicas</a></li>
-                    </ul>
-                  </li>
-                </ul>
-              </li>
-
-              <li>
-                <a href="#managing-apps">Managing apps</a>
-                <ul>
-                  <li>
-                    <a href="#scaling-apps">Scaling apps</a>
-                    <ul>
-                      <li><a href="#increasing-instances">Increasing instances</a></li>
-                      <li><a href="#increasing-memory-and-disk-space">Increasing memory and disk space</a></li>
-                      <li><a href="#more-about-scaling">More about scaling</a></li>
-                    </ul>
-                  </li>
-                  <li>
-                    <a href="#quotas">Quotas</a>
-                    <ul>
-                      <li><a href="#quota-allocations">Quota allocations</a></li>
-                      <li><a href="#quota-limits">Quota limits</a></li>
-                      <li><a href="#application-quotas">Application quotas</a></li>
-                      <li><a href="#memory-share-and-compute-share">Memory share and compute share</a></li>
-                      <li><a href="#application-quota-sizing">Application quota sizing</a></li>
-                    </ul>
-                  </li>
+          <div class="toc" data-module="table-of-contents">
+            <a href="#" class="toc__close js-toc-close">Close</a>
+            <nav class="toc__list" id="toc" aria-labelledby="toc-heading">
+              <ul id="toc" class="js-toc-list">
```

Comment:
It would be handy if this <ul> didn't have any attributes. It would allow a table of contents generator to be completely agnostic of it's context.

This change is making the table of contents PR unmergeable atm.